test/torchaudio_unittest/datasets/datasets_test.py

```diff
@@ -1,7 +1,3 @@
-from torchaudio.datasets.librispeech import LIBRISPEECH
```

Comment:
You can delete the whole file.

Reply:
this line has already been deleted in the first commit.

Reply:
Not sure what you mean by the first commit, but this file no longer contains any test (as planned), so please remove the file itself.

Reply:
test/torchaudio_unittest/datasets/datasets_test.py removed